compiler/ballerina-treegen/src/main/resources/syntax_tree_descriptor.json

```
@@ -693,6 +693,11 @@
                 {
                     "name": "expression",
                     "type": "ExpressionNode"
+                },
+                {
+                    "name": "onFailClause",
```

```suggestion
                    "name": "onFailCheck",
```
As per our last discussion.